tests/integration/cli_did_test.go

```diff
@@ -0,0 +1,106 @@
+//go:build integration
+
+package integration
+
+import (
+	"crypto/ed25519"
+
+	"github.com/cheqd/cheqd-node/tests/integration/cli"
+	"github.com/cheqd/cheqd-node/tests/integration/network"
+	"github.com/cheqd/cheqd-node/tests/integration/testdata"
+	cli_types "github.com/cheqd/cheqd-node/x/cheqd/client/cli"
+	"github.com/cheqd/cheqd-node/x/cheqd/types"
+	"github.com/google/uuid"
+	"github.com/multiformats/go-multibase"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("cheqd cli", func() {
+	It("can create diddoc, update it and query the result", func() {
```

Comment:
These are very different actions and this is an overloaded test. Especially once we have pricing differentiation, where we _do_ care what kind of DID transaction it is.